tests/integration/test_agentd/test_agentd_enrollment_params.py

```diff
@@ -86,6 +95,17 @@ def clean_log_file():
 
 
 def override_wazuh_conf(configuration):
+    """
+    It applies custom settings on ossec.conf file from values located under
+    "configuration" section of tests found in a YAML file. For this purpose,
+    it stops the wazuh-agentd service, applies the settings and starts it again.
+
+    Args:
+        configuration: New parameters to be applied.
```

Comment:
The type is missing here.

Reply:
https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb